config/common_versions.yml

```diff
@@ -1,5 +1,5 @@
-title: v1.1.3
+title: v1.1.4
```

Comment:
Is this also motivated by CVE fixes?

Reply:
Yes, we updated key-cert-provisioner to address CVEs